nextflow.config

```diff
@@ -248,62 +214,11 @@ profiles {
     }
   }
 
-  //
-  // Clemson's Palmetto cluster uses the PBS scheduler. Here we provide
-  // an example for execution of this workflow on Palmetto with some
-  // defaults for all steps of the workflow.
-  //
-  pbs {
-    process {
-      executor = "pbspro"
-      time = "8h"
-      cpus = 2
-      memory = 2.GB
-
-      withLabel:multithreaded {
-        cpus = 8
-        memory = 8.GB
-      }
-    }
-    executor {
-      queueSize = "${params.execution.queue_size}"
-    }
-  }
-
-  //
-  // WSU's Kamiak cluster uses the SLURM scheduler. Here we provide
-  // an example for execution of this workflow on Kamiak with some
-  // defaults for all steps of the workflow.
-  //
-  slurm {
-    process {
-      executor = "slurm"
-      queue = "ficklin"
-      time = "24h"
-      cpus = 1
-
-      withLabel:multithreaded {
-        cpus = 1
-      }
-    }
-    executor {
-      queueSize = "${params.execution.queue_size}"
-    }
-  }
-
-  //
-  // In lieu of using Docker or Singularity, software dependencies
-  // can be provided by Environment Modules (or a compatible equivalent
-  // such as Lmod). Module names may vary from system to system; the
-  // following profiles are specific to Palmetto and Kamiak respectively,
-  // but you will likely need to create your own profile that uses the
-  // correct module names for your cluster.
-  //
-  palmetto_modules {
+  modules_kamiak {
     withLabel:gemmaker {
       module = "fastQC"
       module = "hisat2"
-      module = "anaconda3"
+      module = "python3"
```

Comment:
I know I changed this to python3 in a previous commit.  So why are fixes being rolled back in merges? is it Git or are we not being careful enough?